README.md

```diff
@@ -77,7 +79,34 @@ The fallback option is principally useful for single-page applications (SPAs) wh
 
 The second case is useful if you have multiple SPAs within the one filesystem. e.g., */* and */admin*.
 
+## Docker Usage
```

Comment:
Maybe "Use goStatic with Docker" is a better name? Also, H3 title may be better in the Usage part